### PR TITLE
Monitoring services

### PR DESCRIPTION
**Pull request by aciceri**
Required by #85

### CODE REVIEW

File: nix/host/default.nix [hunk 1]
```diff
@@ -17,6 +17,11 @@ let
           ./scheduled-tasks.nix
           ./docker.nix
           ./nginx.nix
+          ./prometheus.nix
+          ./grafana.nix
+          ./promtail.nix
+          ./loki.nix
+          ./promtail.nix
```
Comment (albertodvp) [inline nix/host/default.nix:24]:
```suggestion
          ./promtail.nix
          ./loki.nix
```

File: nix/host/grafana.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{ config, ... }: {
+  services.grafana = {
+    enable = true;
+    settings = {
+      server = {
+        domain = "status.cardanow.staging.mlabs.city";
+        http_addr = "localhost";
+        http_port = 2342;
+        root_url = "https://${config.services.grafana.settings.server.domain}:443/";
```
Comment (albertodvp) [inline nix/host/grafana.nix:9]: Is it expected to have the trailing `/`? If this works I guess it is

**Review by albertodvp — CHANGES_REQUESTED** [review]
I'm not confident on the application specific config but everything looks reasonable and we can tweak things if we want to change anything

File: nix/host/grafana.nix (hunk 2 above)
Comment (aciceri) [reply nix/host/grafana.nix]: Done

File: nix/host/default.nix (hunk 1 above)
Comment (aciceri) [reply nix/host/default.nix]: Done, thanks

**Pull request merged**